Mapper/BaseGroupedMapper.php

```diff
@@ -257,6 +263,11 @@ public function hasOpenTab()
     abstract protected function setTabs(array $tabs);
 
     /**
+     * @return string
+     */
+    abstract protected function getContext();
```

Comment:
I think this should be commented and added on next major.

Like it's done here: https://github.com/sonata-project/SonataAdminBundle/blob/b369f59375608a4c5f1b5d7fccf1ee6763dd7f9a/Admin/AdminInterface.php#L310-L325